locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -253,8 +253,8 @@ msgstr "Login is required"
 msgid "Login maximum length is %d"
 msgstr "Login maximum length is %d"
 
-msgid "Login must contain only numbers, letters, -, _ or ."
-msgstr "Login must contain only numbers, letters, -, _ or ."
+msgid "Login must contain only numbers, letters, -, _ or . "
```

Comment:
There is whitespace at the end of this string.

Reply:
Same here

Reply:
So the linting error was that I cant a sentence starting with a uppercase char and ending with a full stop. so I interchanged positions of . with _